cfme/tests/containers/test_projects_dashboard.py

```diff
@@ -103,3 +103,24 @@ def test_projects_dashboard_icons(provider, appliance, soft_assert, container_pr
                 containers_cls.__name__, api_values[containers_cls], statusbox_value
             )
         )
+
+
+def test_project_has_provider(appliance, soft_assert):
+    """
+    Test provider name existence in all projects table.
+    Steps:
+      * navigate to all project page
+      * get through all the project to ensure that the provider column isn't
+        empty on each on each of the projects
+    """
+    projects_collection = appliance.collections.container_projects
+
+    all_project_view = navigate_to(projects_collection, "All")
+    all_tables_rows = all_project_view.entities.get_all()
+
```

Comment:
REQUIRED: I'd add assert statement that all_tables_rows isn't empty. Otherwise, this test will test nothing if there isn't projects.

Reply:
@izapolsk Well this is a good point, I am working to this...